kadasrouting/core/datacatalogueclient.py

```diff
@@ -0,0 +1,106 @@
+import os
+import json
+
+from PyQt5.QtCore import (
+    QUrl,
+    QFile,
+    QDir
+)
+
+from PyQt5.QtNetwork import (
+    QNetworkRequest,
+    QNetworkReply
+)
+
+from qgis.core import (
+    QgsNetworkAccessManager
+)
+
+from kadasrouting.utilities import appDataDir, waitcursor
+
+from pyplugin_installer import unzip
+
+
+class DataCatalogueClient():
+
+    NOT_INSTALLED, UPDATABLE, UP_TO_DATE = range(3)
+
+    # TODO
+    DEFAULT_URL = ""
```

Comment:
@volaya so basically here is where we should put `DEFAULT_URL = "https://geoinfo-kadas.op.intra2.admin.ch/portal/sharing/rest/search?q=tags:%22valhalla_network%22&f=pjson"` or am I wrong?

Reply:
or maybe only `DEFAULT_URL = "https://geoinfo-kadas.op.intra2.admin.ch/portal/sharing/rest"`